### DISCUSSION TITLE
Status of the project

### DISCUSSION THREAD

**Discussion by thives** [quicktype/quicktype#2640] [General]
What is the current status of this project? Myself and others have posted issues and PRs, but there are no responses or actions from anyone with privileges.

The CI pipeline seems to be down. I'm no expert, but I believe posting PRs should trigger a build and test automatically. None of that is happening currently.

Is there anything we can do to help?

**Comment by successkrisz** [comment]
For a package which had so many contributors this one seems to be dead at this point, no interactions from the maintainers for a while now 😢

**Reply by kriswest** [reply]
There was a flurry of activity from the maintainers back in May 2024, but its been relatively quiet since.

**Reply by successkrisz** [reply]
Yeah, the schema libraries generated by this project did move on, and this library is now incompatible with those. We are using typescript-effect-schema for example and now pretty much have to replace usage of quicktype due to incompatibility issues 🤷‍♂️

**Comment by goto40** [comment]
Any news?

**Reply by xbc5** [reply]
It looks like both of the maintainers are working in private repositories - both are very active. The API docs were updated last week. So, my guess is that they're working on some business idea, or a new back end together; probably related in some way.
